slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -110,6 +107,7 @@ public DbSlashingProtection(
                 .registerModule(new InterchangeModule())
                 .configure(FLUSH_AFTER_WRITE_VALUE, true)
                 .enable(SerializationFeature.INDENT_OUTPUT));
+    this.pruner = new Pruner(jdbi, signedBlocksDao, signedAttestationsDao, lowWatermarkDao);
```

Comment:
sounds like Pruner should be a generic idea - and we should have a DB specific version of one

Reply:
it's only going to be used for the db slashing protection so don't think this makes sense to be generic at the moment